### ISSUE TITLE
objectAtIndexedSubscript crash

### ISSUE THREAD

**Issue by martin072** [ccgus/fmdb#730]
Hi,

I seem to have an issue that is hard to reproduce, but seems to point in to fmdb.
One function makes some queries to the a database but sometimes crashes with this message:

`-[__NSArrayI objectAtIndexedSubscript:]: index 62 beyond bounds [0 .. 61]`

My code is not using anything containing 'objectAtIndexedSubscript', so a quick search only came with this used in 'FMResultSet'

Any one an idea where this could come from?

**Comment by robertmryan** [comment]
Yes, but you don’t have to use `objectAtIndexedSubscript`. You can get this if you use subscripts at all with arrays. If you look at your stack trace, you should be able to diagnose where you’re exceeding the bounds of your array.

But this is unlikely to be related to FMDB itself.
